gradle-plugins/buildSrc/build.gradle.kts

```diff
@@ -9,4 +9,6 @@ repositories {
 dependencies {
     compileOnly(gradleApi())
     implementation(kotlin("stdlib"))
+    implementation("io.ktor:ktor-client-core:1.5.1")
```

Comment:
Why it is needed?

Reply:
It is for printAllAndroidxReplacements script (fetch all needed dependencies from maven repo).

It is only available for plugin build, and won't be in the plugin itself.